app/Http/Resources/RequestNcpoDetailedResource.php

```
@@ -14,15 +14,29 @@ class RequestNcpoDetailedResource extends JsonResource
             'created_by' => $this->created_by,
-            'new_po_total' => $this->new_po_total,
-            'items' => RequestNcpoItemResource::collection($this->items),
+            'total' => $this->purchaseOrder?->requestCanvassSummary?->grand_total_amount
+                ? number_format($this->purchaseOrder->requestCanvassSummary->grand_total_amount, 2)
+                : '0.00',
+            'new_po_total' => number_format($this->new_po_total, 2),
+            'original' => [
+                'items'    => $this->purchaseOrder?->items,
+            ],
+            'changed' => [
+                'items' => RequestNcpoItemResource::collection($this->items),
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\RequestNcpoItemResource' in method 'toArray'. (undefined)

(StaticAccess)